region/crateria/central.json

```diff
@@ -216,6 +234,12 @@
                 }
               ],
               "unlock": null
+            },
+            {
+              "id": 6,
```

Comment:
My reaction would have been to make the node link always accessible (and to give it a free link back to 5 as well) but to have `ZebesSetAblaze` in as an `unlock`. I don't know if that's better, but it's more consistent with how the rest of the flags are handled.